core/src/test/scripts/system_tests_validate_data.py

```diff
@@ -26,8 +27,17 @@ class ValidateDataSystemTester(unittest.TestCase):
     the html report when requested?", etc)
     '''
 
+    def setUp(self):
+        def dummy_pom_path():
+            return "test_data/test.xml"
+        self.orig_defined_sample_ids = get_pom_path
+        validateData.get_pom_path = dummy_pom_path
+
     def tearDown(self):
         """Close logging handlers after running validator and remove tmpdir."""
+        # restore original function
+        validateData.get_pom_path = self.orig_defined_sample_ids
```

Comment:
Since this doesn't have anything to do with sample IDs, could you give this attribute a different name? Something like `self.orig_pom_path_function`, I mean.